src/chart.ts

```diff
@@ -635,18 +635,63 @@ export class Chart extends EventEmitter.EventEmitter {
   updateAxes() {
     const instance = this
     const canvas = instance.canvas.merge(instance.canvas.enter)
-    canvas.select('.x.axis').call(instance.meta.xAxis)
+
+    // center the axes
+    const yMin = this.meta.yScale.domain()[0]
```

Comment:
Nit: All of these variables are only used inside their corresponding ifs, consider moving them to be inside them so that they're properly scoped.